great_expectations_cloud/agent/agent.py

```diff
@@ -111,6 +120,11 @@ def _listen(self) -> None:
             if subscriber is not None:
                 subscriber.close()
 
+    def _get_current_gx_agent_version(self) -> str:
+        version: str = metadata_version(self._PYPI_GX_AGENT_PACKAGE_NAME)
```

Comment:
Nice! I was going to suggest this (saw the `pkg_resources` version last night).

Reply:
Yeah, this is awesome; didn't realize it was that easy!